user/models.py

```diff
@@ -0,0 +1,27 @@
+# autopep8: off
+from django.db import models
+
+
+class User(models.Model):
+    account         = models.CharField(max_length=45)
+    password        = models.CharField(max_length=600)
+    name            = models.CharField(max_length=45)
+    email           = models.EmailField(max_length=254)
+    phone_number    = models.CharField(max_length=45)
+    date_of_birth   = models.DateField(
+        auto_now=False, auto_now_add=False, null=True)
```

Comment:
```suggestion
    date_of_birth   = models.DateField(auto_now=False, auto_now_add=False, null=True)
```
속성 선언시에는 파라미터가 정말 많지 않은 이상 분리 하지 않으셔도 됩니다 👍

Reply:
한줄로 수정했습니다 :)